packages/portalnetwork/src/client/routingTable.ts

```diff
@@ -2,17 +2,24 @@ import { KademliaRoutingTable } from '@chainsafe/discv5'
 
 import type { ENR, NodeId } from '@chainsafe/enr'
 import type { Debugger } from 'debug'
+import type { IClientInfo, INodeAddress } from '../index.js'
 import { shortId } from '../index.js'
+import { ScoredPeer } from '../networks/peers.js'
 export class PortalNetworkRoutingTable extends KademliaRoutingTable {
   public logger?: Debugger
   private radiusMap: Map<NodeId, bigint>
   private gossipMap: Map<NodeId, Set<Uint8Array>>
   private ignored: [number, NodeId][]
+  /**
+   * Map of all known peers in the network
+   */
+  public networkCensus: Map<string, ScoredPeer>
```

Comment:
```suggestion
  public networkCensus: Map<NodeId, ScoredPeer>
```

Import `NodeId` from `@chainsafe/enr`